### ISSUE TITLE
Online limb-darkening tool has some missing JWST bandpasses

### ISSUE THREAD

**Issue by nespinoza** [ExoCTK/exoctk#406]
Hi folks,

 As pointed out in today's ExoCTK team meeting by @JeffValenti, NIRSpec, NIRISS orders 2/3 and different NIRCam combinations are missing from the possible bandpasses in the online tool. We think this is OK for now, but this will be critical for JWST data analysis. 

 I'm assigning this to @hover2pi as he might have some quick ideas on how to solve this. I believe the easiest thing would be to ask the user to upload a bandpass in a specified format --- but the optimal would be to retrieve bandpasses periodically from some central source (e.g., is there any CRDS file that stores this info for JWST bandpasses?).

 I'm setting this for v1.2 (i.e., our next-next release), as this is low priority for now.

N.

**Comment by hover2pi** [comment]
Yeah, this has been a problem because there is no central repository for the bandpasses of each instrument. I have had to just get some from JDOX and contact the instrument teams directly. I would strongly advocate for these files to be quality controlled and centrally located.

**Comment by hover2pi** [comment]
Possible solution from Klaus Pontoppidan:

```
def get_pce(instrument,mode,config):
   obsmode = {
              ‘instrument’: instrument,
              ‘mode’: mode,
              ‘filter’: config[‘filter’],
              ‘aperture’: config[‘aperture’],
              ‘disperser’: config[‘disperser’]
              }
   conf = {‘instrument’: obsmode}
   i = InstrumentFactory(config=conf)
   wr = i.get_wave_range()
   wave = np.linspace(wr[‘wmin’], wr[‘wmax’], num=500)
   pce = i.get_total_eff(wave)
   return wave,pce
```

**Comment by hover2pi** [comment]
I ended up just writing a function in the `helpers.py` module to grab the FITS files in the `pandeia_refdata` directory and save them locally as text files so `svo_filters` can read them. It didn't work.

So I released `svo_filters v0.3.0` which fixes a few small bugs in the `load_txt` method and updated the relevant code in `limb_darkening_fit.py` (as well as the env dependency) which DID work, but it has exposed something curious.

The NIRISS dispersers all seem to be problematic. The GR700XD has strange linearly interpolated values in their throughput curves and the GR150C/R grisms are just constant values. Is this correct? I checked the headers and there are no instructions for some special conversion or manipulating. The NRM filter is all ones but that is intentional since the ETC separately takes it into account.

The NIRCam and NIRSpec dispersers all look reasonable so it appears to only be the files delivered by NIRISS to the ETC.

Any ideas?

<img width="300" alt="Screen Shot 2020-12-29 at 10 52 22 PM" src="https://user-images.githubusercontent.com/34687/103329369-811a5300-4a2a-11eb-87f3-f71abdfd7a58.png">
<img width="300" alt="Screen Shot 2020-12-29 at 10 52 03 PM" src="https://user-images.githubusercontent.com/34687/103329374-85467080-4a2a-11eb-8d6a-ad6581bd2fe3.png">
<img width="300" alt="Screen Shot 2020-12-29 at 10 50 01 PM" src="https://user-images.githubusercontent.com/34687/103329372-84adda00-4a2a-11eb-8e35-dfb85ca9a64d.png">
<img width="300" alt="Screen Shot 2020-12-29 at 10 51 42 PM" src="https://user-images.githubusercontent.com/34687/103329373-84adda00-4a2a-11eb-825a-c4a5d65d3c97.png">

**Comment by hover2pi** [comment]
Ok, I confirmed that with `pandeia_data` v1.5.3 the NIRISS GR700XD is still messed up as shown above. I'm going to check with the `pandeia` folks to see if they know about it and why it looks the way it does.

**Comment by pontoppi** [comment]
Looks like you are plotting the dispersion (dl/ds) micron per pixel, not the throughput.

**Comment by hover2pi** [comment]
This is resolved in #458
